### PR TITLE
docs: Fix vscode instructions

### LINKED ISSUE #959: Update vscode instructions
@gnikit released a new version of the vscode fortran extension which simplifies the installation process a lot.
We should update our [installation instructions](https://github.com/MODFLOW-USGS/modflow6/blob/develop/.vscode/README.md) accordingly.

### PR DESCRIPTION
**Pull request by Hofer-Julian**
These are the minimal changes to get valid installation instructions for vscode again.
When I find a bit more time, I'll open a bigger follow-up PR to clean up a few things.

For existing installations:

- Uninstall the FORTRAN IntelliSense and Fortran Breakpoint Support extensions.

- Then execute:
```
pip uninstall fortran-language-server
pip install fortls
```

- Then add the fortls path to your workspace settings:
```json
{
    "fortran.fortls.path": "/path/to/fortls"
}
```

Please let me know if you encounter problems.

@jdhughes-usgs, @langevin-usgs, @mjr-deltares, @mjreno, @jmccreight 

Fixes #959

### CODE REVIEW

**Comment by mjr-deltares** [comment]
FYI it's working for me

File: .vscode/README.md [hunk 1]
```diff
@@ -57,22 +54,14 @@ pip install -U fortran-language-server
 - cmd: `where fortls`
 - PowerShell: `Get-Command fortls`
 
-3. Add this to `.vscode/settings.json`. In case this file does not exist yet, create a new one. Also remember to adapt `fortran-ls.executablePath` to the path you get in the step before.
+3. Add the setting "fortran.fortls.path" to `.vscode/settings.json` and set its value to th result of step 2.
```
Comment (langevin-usgs) [inline .vscode/README.md:57]:
```suggestion
3. Add the setting "fortran.fortls.path" to `.vscode/settings.json` and set its value to the result of step 2.
```

**Review by langevin-usgs — CHANGES_REQUESTED** [review]
Yep.  Looks good --- simpler.  Thanks.

File: .vscode/README.md (hunk 1 above)
Comment (Hofer-Julian) [reply .vscode/README.md]: Thanks!

**Pull request merged**